### PR TITLE
update German translation

### PR DESCRIPTION
**Pull request by tiltX**
This updates a few strings that currently exist in the locale files. It doesn't add translations to the new English strings that have been added or renamed since 2.0.8 was released.

Is there a workflow for those? It seems to me as if the missing strings are generated before each release. Is this an automated  process or are you using certain tools/scripts? I'm asking because I'd like to tackle those new strings as well, but it would be a bit of a chore to find all missing keys in the *.de.text files.

### CODE REVIEW

**Comment by tiltX** [comment]
I think such a string freeze would be great! I don't know if a field in the json file would be of much use (at least to me, when checking strings) because there's always git's commit history to see if/what has changed.

There's one thing though that I felt made my work more difficult: you have renamed some keys in the json files but intstead of renaming them in each locale the strings have disappeared completely. I can compare English and German files side-by-side to easily see what's missing, but it's harder to find out what the original translation was (or if the missing key is in fact a new string).

long story short: I think it would be great if renamed keys in the English files could also be renamed in the locales.

**Comment by darbyjohnston** [comment]
Hi sorry about the deleted keys, I'll be more careful about that in the future. Whenever I changed a key or translation I was in the habit of deleting them from the other language files, but if just the key changes that's not necessary and I can rename them instead of deleting them. If the text changes it seems like they should be deleted, otherwise the translated text could be totally different from the English.

I can also start using the GitHub "Discussions" feature to announce new releases and string freezes.
